openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/TemplateOperationsImpl.java

```diff
@@ -139,6 +140,8 @@ public KubernetesList processLocally(ParameterValue... values) {
   }
 
   public KubernetesList processLocally(Map<String, String> valuesMap)  {
+    this.replaceValueStream = new ReplaceValueStream(valuesMap);
```

Comment:
To answer my own question: `It's becase load() may call get() which may call unmarshal`.